examples/local_deployment/substrate.config.yaml

```diff
@@ -1,8 +1,10 @@
+nc-slicer:
+  internalPort: 5000
+  image: 'seelab/substrate-nc-slicer'
+
 data:
   source:
-    - ../data/apple
-
-
+    - /home/zavier/Documents/research/seelab/data/DHCM/input_var
```

Comment:
In cases like this, I usually write this out as `/path/to/my/data/DHCM/input_var` to keep any personal info out of the repo.